chart/assets/operations/instance_groups/api.yaml

```diff
@@ -32,6 +32,27 @@
   path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/ccdb/connection_validation_timeout?
   value: 3600
 
+# Default settings for the CF info endpoints.
+##
+# Note that the version information has to be a plain integer, as per the
+# Membrane::SchemaValidation occuring in `bosh-pre-start-cloud-controller-ng`.
+
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/name?
+  value: 'SUSE CAP Kubecf'
```

Comment:
This would be incorrect for KubeCF (but would be correct for CAP).

Reply:
Tweaked in commit 6997dff1ed9dd3f7eef0472ad4ea5ffe04afb737